packages/client/src/components/ReviewsDisplay/ReviewsDisplay.jsx

```diff
@@ -1,23 +1,57 @@
-import styled from '@emotion/styled';
-import AddCircleIcon from '@mui/icons-material/AddCircle';
-import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
-import ThumbDownOffAltIcon from '@mui/icons-material/ThumbDownOffAlt';
-import ThumbUpIcon from '@mui/icons-material/ThumbUp';
+import React, { useEffect, useState } from 'react';
 import {
-  Avatar,
   Box,
-  Button,
-  Divider,
   List,
   ListItem,
+  Divider,
   ListItemAvatar,
+  Avatar,
   Typography,
+  Button,
+  CircularProgress,
 } from '@mui/material';
-import * as React from 'react';
-import { MovieDetailsLayout } from '../../containers/MovieDetailsLayout/MovieDetailsLayout';
+
+import AddCircleIcon from '@mui/icons-material/AddCircle';
+import ThumbDownOffAltIcon from '@mui/icons-material/ThumbDownOffAlt';
+import ThumbUpIcon from '@mui/icons-material/ThumbUp';
 import { RatingStars } from '../RatingStars/RatingStars';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import ArrowDropUpIcon from '@mui/icons-material/ArrowDropUp';
+import { MovieDetailsLayout } from '../../containers/MovieDetailsLayout/MovieDetailsLayout';
+import styled from '@emotion/styled';
+import { apiURL } from '../../apiURL';
+
+export function ReviewsDisplay({ movieID }) {
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [showAllReviews, setShowAllReviews] = useState(false);
 
-export function ReviewsDisplay() {
+  useEffect(() => {
+    const fetchReviewsList = async () => {
+      setIsLoading(true);
+      if (movieID) {
```

Comment:
I am unsure, but if I was coding this I would more `if(movieId)` to line 30 for early return, so that we don't start any action without the id not being there. Like so:
```
useEffect(() => {
if(!movieID) return;
//rest of your code
}
```

This is just a suggestion though, I'm unsure if you are comfortable with early returns :)

Reply:
yah i will add the early return to do nothing if there is no movie id, i have no problem with early return :)